### DISCUSSION TITLE
请问签到框架是否支持socks代理？

### DISCUSSION THREAD

**Discussion by hecarli555** [qd-today/qd#438] [Q&A]
请问签到框架是否支持socks代理？指令应该怎么写？能否来个模板看看？

**Comment by hecarli555** [comment]
docker run -d \
--name qiandao \
-v /volume1/docker/qiandao/config:/usr/src/app/config \
-e PORT=8582 \
--env HTTP_PROXY="http://192.168.50.2:1081" \
--restart=always \
--network=host \
a76yyyy/qiandao:latest
例如我这个，怎么不生效？

**Reply by a76yyyy** [reply]
文档里部署部分的环境变量里有介绍

**Reply by hecarli555** [reply]
情况是这样的，家里用的物理爱快地址为192.168.50.1，然后爱快里虚拟了op来科学上网，地址为192.168.50.2，签到框架是部署在192.168.50.1下，然后op里用的passwall开启socks代理，端口为1811，然后V2EX签到代理那里填的是http://192.168.50.2:1811,实测V2EX签到失败，请问这是什么原因？

**Reply by a76yyyy** [reply]
地址应该是 socks5h://192.168.50.2:1811
或者 socks5://192.168.50.2:1811

并且你要确定框架能访问到这个地址

**Reply by hecarli555** [reply]
框架是部署在192.168.50.1爱快下的，爱快网段都没有科学上网，所以意思是框架要部署在op的网段下才能用。。。？
